Makefile

```diff
@@ -0,0 +1,85 @@
+.DEFAULT_GOAL:=help
```

Comment:
Could you update the dev guide with how to use the *Makefile*?

Reply:
I added a note about using `make install-local`, but the rest of the Makefile only applies for deploying the operator to the cluster. I still haven't gotten that far.